README.md

```diff
@@ -1,2 +1,24 @@
 # hw01
-Repository for first CFSS homework assignment
```

Comment:
It shows that you deleted the line “Repository for first CFSS homework assignment” but kept “hw01”. Even though the header “hw01” could be removed when using Markdown, it still shows that you already mastered those steps.